.github/workflows/cleanup-runtime.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# FIXME: disable FATS until this test only PR is ready for review
```

Comment:
it may be easier to comment out the fats job in .github/workflows/ci.yaml, but this works too

Reply:
I'll try that as the current approach still executes some unnecessary stuff.